app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    main_list = []
+
+    for person in people:
+        main_list.append(Person(person["name"], person["age"]))
+
+    for person in people:
+        person_now = [p for p in main_list if p.name == person["name"]][0]
```

Comment:
Using single-letter variable names is not appropriate, replace this list comprehension with 
```suggestion
        current_person = Person.people[person["name"]]
```